### PR TITLE
miscellaneous AD metadata updates

### PR DESCRIPTION
**Pull request by avanlinden**
Adding several values to existing terms and creating one new term for multiome data.

### CODE REVIEW

File: terms/ngs/libraryPrep.json [hunk 1]
```diff
@@ -44,6 +44,11 @@
             "description": "RNA library with size selection for microRNAs",
             "source": "Sage Bionetworks"
         },
+	{
+            "const": "mulitome",
```
Comment (tmzintel) [inline terms/ngs/libraryPrep.json:48]: there is a typo here - 'mulitome'

**Review by tmzintel — CHANGES_REQUESTED** [review]
I left a comment on the suggested `libraryPrep.json` changes - there's a typo that should be fixed before accepting changes

File: terms/ngs/libraryPrep.json (hunk 1 above)
Comment (avanlinden) [reply terms/ngs/libraryPrep.json:48]: Oh my goodness, good catch!

**Review by tmzintel — APPROVED** [review]
Looks good!

**Pull request merged**